mitosheet/src/types.tsx

```diff
@@ -67,6 +67,16 @@ export enum UpdateType {
     RenderCountUpdate = 'render_count_update',
     ChecklistUpdate = 'checklist_update',
     UpdateExistingImportUpdate = 'update_existing_import_update',
+    CodeOptionsUpdate = 'code_options_update',
+}
+
+type ParamName = string;
+type ParamValue = string;
+
+export type CodeOptions = {
```

Comment:
Add a ts_type test to make sure these are in sync with the backend?

Reply:
Hard to do with these types.